docs/source-pytorch/upgrade/sections/1_4_advanced.rst

```diff
@@ -8,28 +8,36 @@
 
    * - called ``ModelCheckpoint.save_function``
      - now call ``Trainer.save_checkpoint``
-     - #7201
+     - `PR7201`_
```

Comment:
Why not do
```suggestion
     - `#7201 <https://github.com/Lightning-AI/lightning/pull/7201>`__
```
?

Reply:
in many case one PR is referred in several items